distribution/docs/src/main/resources/_contents/_registry-contents/managing-registry-contents.adoc

```diff
@@ -1,26 +1,532 @@
-// overview - ~ a paragraph
+
+${ddf-registry} contains the base registry components, plugins, sources and interfaces needed for ${branding} to function as a registry connecting multiple nodes.
 
 === ${ddf-registry} Prerequisites
 
 To use the ${ddf-registry}, the following apps/features must be installed:
 
-// a list
-* ${ddf-spatial}
+* <<Managing ${ddf-spatial},${ddf-spatial}>>
 
 === Installing ${ddf-registry}
 
+* Navigate to the ${admin-console}.
+* Click the *Manage* applications button.
+* Click the *Install* icon. icon:play[]
+* ${ddf-registry} will move to *Active Applications* upon startup.
+
 === Configuring ${ddf-registry}
 
+.Registry Configuration Tab
+[cols="1,1m,1" options="header"]
+|===
+|Name
+|Configuration ID
+|Description
+
+|<<Registry_Federation_Admin_Service,Registry Federation Admin Service>>
+|Registry_Federation_Admin_Service
+|Registry Federation Admin Service Settings
+
+|<<org.codice.ddf.registry.policy.RegistryPolicyPlugin,Registry Policy Plugin>>
+|org.codice.ddf.registry.policy.RegistryPolicyPlugin
+|Registry Policy Plugin Settings
+
+|<<Registry_Configuration_Event_Handler,Registry Source Configuration Handler>>
+|Registry_Configuration_Event_Handler
+|Configurable values for the registry source configuration handler
+
+|<<Csw_Registry_Store,Registry Store>>
+|Csw_Registry_Store
+|Registry CSW Store Settings
+
+|===
+
+.[[Registry_Federation_Admin_Service]]Registry Federation Admin Service
+[cols="1,1m,1,2,1,1" options="header"]
+|===
+|Name
+|Property
+|Type
+|Description
+|Default Value
+|Required
+
+|Registry Sub Poller Interval
+|registrySubPollerInterval
+|String
+|The polling interval for the registry subscriptions. In seconds.
+|30
+|yes
+
+|===
+
+.[[org.codice.ddf.registry.policy.RegistryPolicyPlugin]]Registry Policy Plugin
+[cols="1,1m,1,2,1,1" options="header"]
+|===
+|Name
+|Property
+|Type
+|Description
+|Default Value
+|Required
+
+|Registry CUD Attributes
+|writeAccessPolicyStrings
+|String
+|Roles/attributes required for CUD (create/update/delete) operations on registry entries. Example: {role=role1;type=type1}
```

Comment:
Do we want to use the more familiar acronym `CRUD` for create/read/update/delete operations? Or does reading have no place here?